core/src/test/kotlin/com/malinskiy/marathon/execution/queue/TestResultReporterTest.kt

```diff
@@ -22,6 +26,7 @@ import org.jetbrains.spek.api.dsl.on
 import java.io.File
 
 object TestResultReporterSpec : Spek({
+    initKoin()
```

Comment:
it is not needed with constructor injection

Reply:
not sure how it would work then - this object is created by the test runner